src/Exception/InvalidRecordInSet.php

```diff
@@ -0,0 +1,35 @@
+<?php
+declare(strict_types=1);
+
+namespace Kafka\Exception;
+
+use Kafka\Exception;
+use function sprintf;
+
+final class InvalidRecordInSet extends Exception
+{
+    public static function missingTopic(): self
+    {
+        return new self('You have to set "topic" to your message.');
+    }
+
+    public static function nonExististingTopic(string $topic): self
+    {
+        return new self(sprintf('Requested topic "%s" does not exist. Did you forget to create it?', $topic));
+    }
+
+    public static function topicIsNotString(): self
```

Comment:
It hurts me the fact we have to do this 😢 

Maybe add a `FIXME: kill with 🔥` 😂